Source/WebCore/css/CSSPrimitiveValue.h

```diff
@@ -411,6 +417,16 @@ inline bool isValueID(const CSSPrimitiveValue* value, CSSValueID id)
     return valueID(value) == id;
 }
 
+inline bool isValueID(const RefPtr<CSSPrimitiveValue>& value, CSSValueID id)
```

Comment:
It's not the role of `CSSPrimitiveValue` to know about all the different kinds of pointers/refs in the codebase. This should be handled at caller site maybe?

Reply:
I am not sure I agree with that sentiment. Overloading here makes the call sites much easier to read.